src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_test.py

```diff
@@ -377,9 +377,9 @@ def test_basic_fuzz(self, mock_stdout):
                   80,
               'merge_edge_coverage':
                   1769,
```

Comment:
I say this nearly every time someone changes a stat but these tests really need to be refactored so that adding/changing a stat doesn't require fixing 10 tests.

Reply:
yes need to think more.